Dupla colaborativa: Olimpia Coria- Dianyela Maldonado

DianyelaMaldonado opened the pull request:
Proyecto: Data lovers.
App: On-Rankeds.
 Información tomada de los datos propuestos para el proyecto, en este caso: League of legends.
Laboratoria.
Sede:CDMX.

DianyelaMaldonado committed:
cambios realizados en index.html style.ccs main.js data.js

oo-li committed:
cambios

DianyelaMaldonado committed:
cambios realizados en style.css

oo-li committed:
src/style.css

oo-li committed:
cambioooos

DianyelaMaldonado committed:
probandocambios con colaboradora

DianyelaMaldonado committed:
Se hicieron cambios en style.css se corrigieron conflictos, se modificó archivo readme.md y se gregaron imagenes a la carpeta readme

DianyelaMaldonado committed:
Cambios en README.md se cambió información en el link de las imagenes del prototipo final

DianyelaMaldonado committed:
se agregaron cambios en README.md en imagenes de prototipo final, se habia agregado una letra de más, por eso no se reflejaban al querer verlas

DianyelaMaldonado committed:
En README.md en la línea 13 hasta la 18 de manifiesto se editó información

DianyelaMaldonado committed:
Se añadió informaciónen README.md en cuanto a historias de usuario y sus respectivas tareas.

DianyelaMaldonado committed:
Se añadieron cambios al archivo README.md, salto de línea

DianyelaMaldonado committed:
cambios realizados en Readme.md se añadió imagen prototipo en baja calidad 2

DianyelaMaldonado committed:
Seañadió cambio en Readme.md nombre de imagen

DianyelaMaldonado committed:
cambios en readme.md

DianyelaMaldonado committed:
Se realizaron cambios en descripcion, archivo README.md

DianyelaMaldonado committed:
cambios en README.md  finalizando

DianyelaMaldonado committed:
Se agregó en README.md contenido de objetivos de aprendizaje.

DianyelaMaldonado committed:
se agregaron cambios en README.md se editaron detalles en identación de las palabras

DianyelaMaldonado committed:
Merge pull request #1 from DianyelaMaldonado/feature/Cambios-readme

Feature/cambios readme

DianyelaMaldonado committed:
se agregaron cambios en style.css cambios en botones se cambió la imagen del backgraund se anexó logo de app cambios en index.html botones

DianyelaMaldonado committed:
se agregó logo faltante de la app en readme.md

DianyelaMaldonado committed:
Merge pull request #2 from DianyelaMaldonado/feature/Cambios-readme

se agregó logo faltante de la app en readme.md

DianyelaMaldonado committed:
cambios en carpetas index.html y style.css trabajando con responsive para tablet y movil

DianyelaMaldonado committed:
Merge branch 'master' of https://github.com/DianyelaMaldonado/CDMX009-Data-Lovers

DianyelaMaldonado committed:
cambis realizados en index.html, se añadió información para slider, y style.css se le dieron estilos al slider, en main.js se hizo la funcion a los botones next y prev para la funcionalidad del slider

DianyelaMaldonado committed:
Merge pull request #3 from DianyelaMaldonado/feature/slider

cambios realizados en index.html, se añadió información para slider, y…

DianyelaMaldonado committed:
cambios realizados en style.css en cuanto al responsive de la app, falta botonera de filtrar

DianyelaMaldonado committed:
Merge pull request #4 from DianyelaMaldonado/hotfix/ajustes-responsive

cambios realizados en style.css en cuanto al responsive de la app, fa…

DianyelaMaldonado committed:
se añadió información en carpeta index.html section bottons, se habían borrado

DianyelaMaldonado committed:
se añadieron imagenes en carpeta de readme, capturas muestra, referentes a como quedó el responsive en tablets y móvil

DianyelaMaldonado committed:
cambios en readme.md se añadió descripción y ejemplos en imagenes de responsive en tablet y movil

DianyelaMaldonado committed:
se agregaron cambios a readme.md se cambió informacion en nombre de las imagenes para probar que salgan correctamente y se acomodó la posición del logo.

DianyelaMaldonado committed:
se cambió nombre de imagen para hacer prueba de carga en readme.md

DianyelaMaldonado committed:
cambios realizados en redme.md siguen pruebas para visualización de imagen

DianyelaMaldonado committed:
cargando foto en archivo readme.md pruebas

DianyelaMaldonado committed:
cambios en readme.md se cambio el nombre del merch para probar si por eso no se veía la imagen

DianyelaMaldonado committed:
en readme.md se cambio el link de imagen en la carpeta readme se añadió imagen con formato jpg probando

DianyelaMaldonado committed:
Merge pull request #5 from DianyelaMaldonado/feacture/descripcion-responsive-readme

Feacture/descripcion responsive readme

DianyelaMaldonado committed:
se añadieron cambios en readme.md intentando cargar imagen

DianyelaMaldonado committed:
Merge pull request #6 from DianyelaMaldonado/feacture/descripcion-responsive-readme

se añadieron cambios en readme.md intentando cargar imagen

DianyelaMaldonado committed:
se añadieron cambios en data.js y en index.html la función del filtro totalmente funcional

DianyelaMaldonado committed:
Merge pull request #7 from DianyelaMaldonado/feature/filtros

se añadieron cambios en data.js y en index.html la función del filtro…

DianyelaMaldonado committed:
se añadió cambio en readme.md se cambió en enlace de la imagen del prototipo en baja 2

DianyelaMaldonado committed:
se agregaron cambios en dat.js en links de imagenes para la funcionalidad en responsive, visible al 100% en desktop, tablets y móvil

DianyelaMaldonado committed:
Merge pull request #8 from DianyelaMaldonado/feature/probando-link-api

se agregaron cambios en dat.js en links de imagenes para la funcional…

DianyelaMaldonado committed:
se agrego nueva línea de código al archivo data.js

DianyelaMaldonado committed:
Merge pull request #9 from DianyelaMaldonado/feature/probando-link-api

se agrego nueva línea de código al archivo data.js

DianyelaMaldonado committed:
probando carpeta docs

DianyelaMaldonado committed:
cambios en data.js docs https

DianyelaMaldonado committed:
se eliminó carpeta docs

DianyelaMaldonado committed:
Merge pull request #10 from DianyelaMaldonado/gh-pages

se añadió letra  S al http en data.js

DianyelaMaldonado committed:
se añadió en redme.md link de git hub pages con la app funcional

DianyelaMaldonado committed:
se corrigió información en readme.md con respecto al link de la app funcional

DianyelaMaldonado committed:
Merge pull request #11 from DianyelaMaldonado/feacture/readme-cambio-final-link

Feacture/readme cambio final link

AdrianaHY commented on docs/data.js:
```diff
@@ -43,7 +43,7 @@ function cargarJSON() { //esta función hace la conexión y definimos como quere
 
 function cargarJSONdos(nombreCampeon) {
     let onlyChampion = document.querySelector('#onlyChampion');
```
Por qué se hacen dos peticiones?

AdrianaHY commented on src/data.js:
```diff
@@ -32,42 +30,26 @@ function cargarJSON() { //esta función hace la conexión y definimos como quere
             }
             cardsprueba.innerHTML = html;
-            //console.log(html);
         })
```
Muy buen trabajo! Solo me queda una duda, por qué la interacción con el dom está en este archivo?

AdrianaHY commented on src/data.js:
```diff
@@ -6,7 +6,7 @@ document.addEventListener("DOMContentLoaded", function() {
-    fetch('http://localhost:5000/data/lol/lol.json')
+    fetch('http://localhost:64127/data/lol/lol.json')
         .then(res => {
             return res.json()
```
Por qué hacemos una llamada al localhost?

AdrianaHY commented on src/index.html:
```diff
@@ -1,14 +1,76 @@
+        </header>
+        <!-- slider  -->
+
+        <section id="top">
```
Borrar espacions y comentarios innecesarios.

AdrianaHY commented on src/index.html:
```diff
@@ -1,14 +1,76 @@
+                        <li> <button class="filter" data-type="Tank"> TANK<img class="b6" src="imagenes/tank2.png"> </button> </li>
+                        <li> <button class="filter" data-type="All"> ALL CHAMPS<img class="b7"> </button> </li>
+                    </ul>
+                </nav>
```
Hay que ser más consistentes con los espacios.

AdrianaHY reviewed (commented):
Chicas! es un excelente trabajo, continúen así! Recuerden optimizar las imágenes para que no sean tan pesadas.

DianyelaMaldonado committed:
I created a new helpers folder where I separated the functions that were together in data.js, the code is modularized.

DianyelaMaldonado committed:
In file data.js
main.js load jsonone.js filterchamps.js variables that were written with var to let were replaced, and line 5 of filterchamps.js declared filteredChamps.

DianyelaMaldonado committed:
Some dependencies in package.json were removed to test

DianyelaMaldonado committed:
add changes in readme.md

DianyelaMaldonado committed:
Merge pull request #12 from DianyelaMaldonado/feature/new-changes--code-restructuring

Feature/new changes  code restructuring

DianyelaMaldonado committed:
move position of image in readme.md

DianyelaMaldonado committed:
Merge pull request #13 from DianyelaMaldonado/feature/new-changes--code-restructuring

move position of image in readme.md

DianyelaMaldonado committed:
Merge pull request #14 from DianyelaMaldonado/master

Master
